airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl/ConnectorServiceTypeControl.tsx

```diff
@@ -122,14 +124,20 @@ const ConnectorServiceTypeControl: React.FC<ConnectorServiceTypeControlProps> =
   const { formatMessage } = useIntl();
   const { openModal, closeModal } = useModalService();
   const { trackMenuOpen, trackNoOptionMessage, trackConnectorSelection } = useAnalyticsTrackFunctions(formType);
+  const reveal = useClearbitReveal();
 
   const workspace = useCurrentWorkspace();
+  const revealSort = useExperiment("connector.revealSort", true);
```

Comment:
I think the default value here should be `false` and not `true`. E.g. in OSS otherwise we'd always try to use this sorting, while we know it actually won't have any clearbit data in OSS.